lib/classes/PluginManager.js

```diff
@@ -148,7 +152,27 @@ class PluginManager {
     if (pluginsObject.localPath) {
       module.paths.unshift(pluginsObject.localPath);
     }
-    this.loadPlugins(pluginsObject.modules);
+    this.loadPlugins(pluginsObject.modules
+      .filter(name => name !== '@serverless/enterprise-plugin'));
```

Comment:
yes.

Reply:
Is it because we don't want to load plugins twice? In that case we already have (or had) that logic in the plugin manager.

Maybe the logic related to the `@serverless/enterprise-plugin` is an edge case though...

Reply:
It's probably good to treat it separately here, but I guess it'll be good to show some warning to the user, that `@serverless/enterprise-plugin` should not be listed as plugin (and carry on as normal).

That way we don't leave an impression that such configuration of a plugin is expected.